pkg/pillar/hypervisor/kvm.go

```diff
@@ -626,6 +626,31 @@ func (ctx kvmContext) GetDomsCPUMem() (map[string]types.DomainMetric, error) {
 	}
 
 	for dom, pid := range ctx.domains {
+		//Fetch cpu usage from top output
+		//FIXME: Once we migrate to kvmtop util, remove this logic
+		cmdOutputBytes, err := wrap.Command("/usr/bin/top", "-n", "1").CombinedOutput()
+		if err != nil {
+			return res, err
+		}
```

Comment:
I'd rather not bail at this point, but fill the CPU portion with 0 as it used to be (to a point where it may be useful to flip the logic in the memory account as well -- make sure that if we have useful CPU statistics we report that even if gathering memory accounting fails)

Reply:
address this point.